src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/MethodQueryIT.java

```diff
@@ -47,8 +47,8 @@ public void queryTest() throws IOException {
         final String result = explainQuery(String.format(Locale.ROOT,
                 "select address from %s where q= query('address:880 Holmes Lane') limit 3",
                 TestsConstants.TEST_INDEX_ACCOUNT));
-        Assert.assertThat(result,
-                containsString("query_string\":{\"query\":\"address:880 Holmes Lane"));
+        Assert.assertThat(result.replaceAll("\\s", ""),
```

Comment:
I recommend instead of doing substring searches, just parse result into JSONObject and get value using patch and compare it (like it is done in most of the codebase). Those removals of spaces are more than messy